package/CHANGELOG

```diff
@@ -13,6 +13,22 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
+04/06/21 amdnsr
```

Comment:
There's already a 2.0.0 entry below, you should only need to add your entry there.

Reply:
Apologies, I did not get it. Should I add my changes just above the following line?
09/05/19 IAlibay, richardjgowers

  * 0.20.1

Reply:
No, there's already an existing 2.0.0 entry just below, this is where it should go.